config.yaml

```diff
@@ -2,4 +2,8 @@ options:
   credentials:
     type: string
     default: ""
-    description: "Space separated username and password for basic authentication."
\ No newline at end of file
+    description: "Space separated username and password for basic authentication."
+  dhparam:
+    type: string
+    default: "/etc/nginx/dhparam/dhparam.pem"
```

Comment:
When using different dhparameters for different hosted sites on the same machine